docs/overview/component/mirror-gateway.md

```diff
@@ -0,0 +1,78 @@
+# Vald Mirror Gateway
+
+Vald Mirror Gateway is an optional component of the Vald, allowing the vector data to be synchronized across multiple Vald clusters.
+
+This component makes it possible to enhance availability during a cluster failure.
+
+<img src="../../../assets/docs/overview/component/mirror-gateway/mirror-gateway.png">
+
+## Responsibility
+
+Vald Mirror Gateway is responsible for the followings:
+
+- Forward user requests ([Insert](https://vald.vdaas.org/docs/api/insert/) / [Upsert](https://vald.vdaas.org/docs/api/upsert/) / [Update](https://vald.vdaas.org/docs/api/update/) / [Remove](https://vald.vdaas.org/docs/api/remove/)) to the other Vald Mirror Gateways in the same group.
+- Manage so that each index state will be the same.
```

Comment:
I think the word manage needs to take `object`.

Something like below?
`Manages the state of indexes stored in all clusters to ensure they are consistent.`

Reply:
```suggestion
- Manages the state of indexes stored in all clusters to ensure they are consistent.
```

Reply:
Thank you. It looks good to me 🙏 
I will fix base sentence based on your suggestion.

Reply:
```suggestion
- Manages the state of indexes stored in all clusters to ensure they are consistent.
```